### PR TITLE
Update supported-code-changes-csharp.md

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by PRMerger4** [comment]
@pranavkm : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by pranavkm** [comment]
FYI @DamianEdwards

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off Thanks!

**Comment by Johnnice21** [comment]
Have you been into bitcoin mining before

File: docs/debugger/supported-code-changes-csharp.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ The table below shows the changes that may be made to C# and Visual Basic code d
 
 ## Unsupported app scenarios
 
-Unsupported apps and platforms include Silverlight 5 and Windows 8.1. Unsupported scenarios in ASP.NET and ASP.NET Core include editing _.aspx_, _.ascx_, _.cshtml_, and _.razor_ files.
+Unsupported apps and platforms include Silverlight 5 and Windows 8.1. Unsupported scenarios in ASP.NET and ASP.NET Core include editing _.aspx_, _.ascx_, _.cshtml_, and _.razor_ files. Editing _.cshtml_, and _.razor_ files is supported in .NET 6 and newer apps.
```
Comment (GitHubber17) [inline docs/debugger/supported-code-changes-csharp.md:83]:
```suggestion
Unsupported apps and platforms include Silverlight 5 and Windows 8.1. Unsupported scenarios in ASP.NET and ASP.NET Core include editing _.aspx_, _.ascx_, _.cshtml_, and _.razor_ files. Editing _.cshtml_, and _.razor_ files is supported in .NET 6 and later apps.
```

**Comment by PRMerger3** [comment]
@GitHubber17 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**